src/services/fields.ts

```diff
@@ -128,6 +136,34 @@ export function getParserForField(fieldName: string, sceneRef: SceneObject): Par
   return parser;
 }
 
+export function getDetectedFieldProps(
```

Comment:
Seeing how this is defined and used, I would argue that, if the parser is json or mixed, it should return a path, or undefined. 

```suggestion
export function getPathForField(
```

Or if you need the parser somewhere:

```suggestion
export function getParserAndPathForField(
```